### PR TITLE
Update phi-3-tutorial.md

### PR DESCRIPTION
**Pull request by parinitarahi**
Change the model path to a variable

### CODE REVIEW

File: examples/python/phi-3-tutorial.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ The `-l` argument is the length of output you would like to generate with the mo
 
 ```bash
 curl https://raw.githubusercontent.com/microsoft/onnxruntime-genai/main/examples/python/phi3-qa.py -o phi3-qa.py
-python phi3-qa.py -m Phi-3-mini-128k-instruct-onnx/directml/directml-int4-awq-block-128 -l 2048
+python phi3-qa.py -m {modelPath}  -l 2048
```
Comment (natke) [inline examples/python/phi-3-tutorial.md:78]: You'll want to change the text above too - to refer to the variable rather than the DML model

Comment (kunal-vaishnavi) [inline examples/python/phi-3-tutorial.md:78]:
```suggestion
python phi3-qa.py -m /path/to/folder/containing/onnx/model/  -l 2048
```

Comment (kunal-vaishnavi) [inline examples/python/phi-3-tutorial.md:78]:
```suggestion
model_path="./Phi-3-mini-128k-instruct-onnx/directml/directml-int4-awq-block-128"
python phi3-qa.py -m $model_path  -l 2048
```

Comment (kunal-vaishnavi) [inline examples/python/phi-3-tutorial.md:78]:
```suggestion
model_path="./Phi-3-mini-128k-instruct-onnx/directml/directml-int4-awq-block-128"
python phi3-qa.py -m $model_path -l 2048
```

**Review by kunal-vaishnavi — APPROVED** [review]

**Review by yufenglee — APPROVED** [review]

**Pull request merged**